main.py

```diff
@@ -121,20 +125,21 @@ def add_admins(message: Message):
             msg = bot.send_message(message.chat.id, "You're not a moderator❌")
         autodelete(bot, msg, 2)
         autodelete(bot, message, 1)
+    else:
+        bot.send_message(message.chat.id, parse_mode="HTML", text="<b>Admins can be added ONLY in our group,\n\
+            \nJoin ➤ <a href='https://telegram.me/joinchat/Bn4ixj84FIZVkwhk2jag6A'>pyTelegramBotAPI</a></b>")
 
 @bot.message_handler(commands=['help'])
 def help_func(message):
-    if message.chat.id == GROUPID:
-        msg = bot.send_message(message.chat.id, "Usage:\n/notes - list of notes\n/add_note - add new note\n/delete_note - delete note\n/get - get note\n/add_admin - add new moderator")
-        autodelete(bot, msg, 2)
-        autodelete(bot, message, 0)
+    msg = bot.send_message(message.chat.id, "Usage:\n/notes - list of notes\n/add_note - add new note\n/delete_note - delete note\n/get - get note\n/add_admin - add new moderator")
+    autodelete(bot, msg, 2)
+    autodelete(bot, message, 0)
```

Comment:
```suggestion
    if message.chat.id == GROUPID: autodelete(bot, message, 0)
```

Reply:
What about `autodelete(bot, msg, 2)` in `Line 135`?

Reply:
line 135 will delete bot's message, not user's message. so no issue

Reply:
Oh, I see...